services/payments/live_solana_test.go

```diff
@@ -24,16 +24,23 @@ import (
 	must "github.com/stretchr/testify/require"
 )
 
+const (
+	splMintAddress  string = "AH86ZDiGbV1GSzqtJ6sgfUbXSXrGKKjju4Bs1Gm75AQq" // SPL mint address on devnet
+	splMintDecimals uint8  = 8                                              // SPL mint decimals on devnet
+)
```

Comment:
Nit: make these `const` local to the test, as they are not used anywhere else. The narrower the scope, the better. Global state must be avoided, even in tests.

Reply:
Agree in principle, but there will be/are more tests in this file that need these values so I think it makes sense to keep them.

Reply:
On second thought, with the addition of some helper functions we are able to achieve what you suggest. :)